spec/spec_helper.rb

```diff
@@ -1,7 +1,9 @@
-require 'simplecov'
-SimpleCov.start 'rails' do
-  add_filter '/config/'
-  add_filter '/lib/rspec/formatters/user_flow_formatter.rb'
+if ENV['CI']
```

Comment:
interesting idea! we are using SimpleCov here for code climate test reporter (which we only run in Travis) so that was my thinking with `CI`...I am torn.